Allow HTTP/2 max concurrent stream configuration (#8781)
traefik/traefik: docs/content/routing/entrypoints.md

    entryPoints:
      name:
        address: ":8888" # same as ":8888/tcp"
        http2:

>           maxConcurrentStreams: 250

Reviewer: ```suggestion
          maxConcurrentStreams: 42
```

    [entryPoints]
      [entryPoints.name]
        address = ":8888" # same as ":8888/tcp"
        [entryPoints.name.http2]

>           maxConcurrentStreams = 250

Reviewer: ```suggestion
          maxConcurrentStreams = 42
```

    ```bash tab="CLI"
    ## Static configuration
    --entryPoints.name.address=:8888 # same as :8888/tcp

>     --entryPoints.name.http2.maxConcurrentStreams=250

Reviewer: ```suggestion
    --entryPoints.name.http2.maxConcurrentStreams=42
```

    Full details for how to specify `address` can be found in [net.Listen](https://golang.org/pkg/net/#Listen) (and [net.Dial](https://golang.org/pkg/net/#Dial)) of the doc for go.

### HTTP/2

#### `maxConcurrentStreams`

_Optional, Default=250_

> `maxConcurrentStreams` defines the number of maximum concurrent streams for HTTP/2.

Reviewer 2: ```suggestion
`maxConcurrentStreams` specifies the number of concurrent streams per connection that each client is allowed to initiate.
```
